README.md

```diff
@@ -106,17 +108,31 @@ This option may be required if you are using a self-signed certificate, an expir
 > NOTE: Starting in plugin version `0.5.0`, this option has no effect in versions of Logstash older than `8.1.0`.
 > If this functionality is required, it is recommended to update Logstash or stay at plugin version `0.4.x` or older.
 
+### `proxy`
+
+* Value type is [string](https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#string) or [hash](https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#hash)
+* Optional
+* No default value
+
+This setting allows to configure an HTTP proxy to route exported data.
+
+The supported configuration options and their syntay are the same as for the [proxy option](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-http.html#plugins-outputs-http-proxy) in the HTTP output plugin:
+
+1. `proxy => http://example.org:1234`
+2. `proxy => { host => "example.org", port => 80, scheme => 'http', user => 'username@host', password => 'password' }`
+3. `proxy => { url =>  'http://example.org:1234', user => 'username@host', password => 'password' }`
```

Comment:
Imo commas does not work. What I've tested is just (3) but without commas and that has worked.
I've triad (1) but that didn't work.
Imo we should add (1) and (2) if we test that only.

Reply:
I took this over from https://www.elastic.co/guide/en/logstash/current/plugins-outputs-http.html#plugins-outputs-http-proxy which is also in line with what they document at https://github.com/logstash-plugins/logstash-mixin-http_client/blob/main/README.md?plain=1#L58-L60.
At https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#hash they indeed document their hash structure to be delimited by spaces, not commas.
I'll take a more thorough look and try it out, thanks!

Reply:
Fixed in https://github.com/dynatrace-oss/logstash-output-dynatrace/commit/d75ff19a527a45caed43a76b79850ac14949d783.